tap_bing_ads/reports.py

```diff
@@ -32,7 +32,13 @@
         'Ctr',
         'Impressions'
     ],
-    'GoalsAndFunnelsReport': ['Goal']
+    'GoalsAndFunnelsReport': ['Goal'],
+    'AgeGenderAudienceReport': [
+        'AccountName',
+        'AdGroupName',
+        'AgeGroup',
+        'Gender'
```

Comment:
@namrata270998 From the bing docs, I also see 'TimePeriod' as the required key. However, I don't see the same mentioned in this list

Reply:
Yes, that's correct. However, it is also a common field for all the reports, hence it is added in another `REPORT_REQUIRED_FIELDS ` list which can be found in this line
https://github.com/singer-io/tap-bing-ads/pull/86/files#diff-5215cbddcb33118eb2c3a413c2ded5a5a94bc52f7b1c7fa88131ba2dcba85684R14